src/ophyd_async/epics/signal/__init__.py

```diff
@@ -1,5 +1,8 @@
 from ._common import LimitPair, Limits, get_supported_values
-from ._p4p import PvaSignalBackend
+
+# Import from _epics_transport so that we get a NotImplemented shim in
+# the case where p4p is not available.
+from ._epics_transport import PvaSignalBackend
```

Comment:
@abbiemery do you know why `PvaSignalBackend` is in our public interface? Maybe for the fastcs signal creation...